toolchain/parser/parser.cpp

```diff
@@ -491,7 +491,7 @@ auto Parser::HandleBraceExpressionParameterError(StateStackEntry state,
   bool can_be_type = kind != BraceExpressionKind::Value;
   bool can_be_value = kind != BraceExpressionKind::Type;
   emitter_->Emit(*position_, ExpectedStructLiteralField,
-                 can_be_type ? "`.field: type`" : "",
+                 can_be_type ? "`.field: Type`" : "",
```

Comment:
This change seems backwards to me; the error may not be great, but capitalizing `Type` I don't think is reducing the ambiguity, it just makes it inconsistent with `field` and `value`.